core_impl/src/test/java/com/google/instrumentation/stats/StatsManagerImplTest.java

```diff
@@ -84,16 +85,24 @@ public void testGetNonexistentView() throws Exception {
   @Test
   public void testRecord() {
     statsManager.registerView(RpcConstants.RPC_CLIENT_ROUNDTRIP_LATENCY_VIEW);
+    Timestamp start = Timestamp.fromMillis(System.currentTimeMillis());
     for (double val : Arrays.<Double>asList(10.0, 20.0, 30.0, 40.0)) {
       statsManager.record(
           oneTag, MeasurementMap.of(RpcConstants.RPC_CLIENT_ROUNDTRIP_LATENCY, val));
+      // Simulate transmission delay.
+      try {
+        Thread.sleep(500);
```

Comment:
We should use a fake clock instead, to keep the tests deterministic and fast.  Also, I think that any refactoring to use a fake clock should be done in a separate PR, because it would be a large change.